code/modules/clothing/glasses/shades.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/clothing/glasses/hud/health/shades
+	name = "Augmented Health Scanner Shades"
+	desc = "A heads-up display that scans the humans in view and provides accurate data about their health status."
+	icon_state = "medshades"
+	item_state = "medshades"
+	spawn_blacklisted = TRUE
+	matter = list(MATERIAL_PLASTIC = 1, MATERIAL_GLASS = 1)
+
+/obj/item/clothing/glasses/hud/security/shades
+	name = "Augmented Security Shades"
+	desc = "A heads-up display that scans the humans in view and provides accurate data about their ID status and security records."
+	icon_state = "secshades"
+	item_state = "secshades"
+	spawn_blacklisted = TRUE
+	matter = list(MATERIAL_PLASTIC = 1, MATERIAL_GLASS = 1)
+
+/obj/item/clothing/glasses/shades
+	spawn_blacklisted = TRUE
+
```

Comment:
Add:
Bad_types = /obj/item/clothing/glasses/shades

Reply:
What wrong about this?

Reply:
Can you normally explain what i need to change?